happy/HappyNetworkRoute.py

```diff
@@ -235,12 +235,13 @@ def __update_state(self):
 
     def run(self):
         with self.getStateLockManager():
-
             self.__pre_check()
 
         self.__configure_nodes_routes()
-
-        self.__configure_gateway_routing()
+        # If device is TAP device, happy will not configure gateway routing
+        # because it will be configured in lwip stack in weave.
```

Comment:
Comment is slightly incorrect.   For TAP devices, addresses and routes will be configured by LwIP stack in *whatever process LwIP is running*

Reply:
OK, will fix